src/SimulationRunner.cc

```diff
@@ -428,8 +428,11 @@ void SimulationRunner::PublishStats()
 
   if (this->Stepping())
   {
+    // Remove this header in Gazebo H
```

Comment:
It's usually helpful to have the word "deprecated" around the comment so we can grep for it later